### ISSUE TITLE
run error

### ISSUE THREAD

**Issue by Wcs200369** [xiaoyeli/superlu_dist#158]
When i run audikw_1.mtx on superlu_dist with 32 rank,  i get an assert : pdgstrf: Assertion `nbrow*(st_col+num_col_stream) < buffer_size' failed. what is this mean ang how can i fix it

**Comment by xiaoyeli** [comment]
The default size (256MB) to use GPU memory exceeded. You can increase it to something a lot larger by setting the environment variable:
export SUPERLU_MAX_BUFFER_SIZE=1024000000

This paper describes the user-tunable parameters, esp. for GPU:
https://dl.acm.org/doi/full/10.1145/3577197

**Comment by jhlee508** [comment]
This worked for me.
https://github.com/xiaoyeli/superlu_dist/blob/eac44cf48878f8699cc19fb566832b6736596727/SRC/double/dSchCompUdt-gpu.c#L219

The line in the code here should be fixed as below:
```bash
 assert(nbrow*(st_col+num_col_stream) <= buffer_size);
```

**Comment by xiaoyeli** [comment]
I don't think this is important. But I pushed a change in master branch. It will be in the next release.
